src/features/group/components/GroupList.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link';
+
+import { IGroup } from '../types/definition';
+import GroupItem from './GroupItem';
+export default async function GroupList({ groups }: { groups: IGroup[] }) {
+  return (
+    <>
+      {groups.map((group) => (
+        <Link href={`/groups/${group.groupId}`} key={group.groupId}>
```

Comment:
You can move the `<Link>` tags into the `<GroupItem>`.

Reply:
That make sense, I fixed.